geniesp/bpc_config.py

```diff
@@ -112,7 +112,7 @@ class Bladder(BpcProjectRunner):
     # Sponsorted project name
     _SPONSORED_PROJECT = "BLADDER"
     # Redcap codes to cbioportal mapping synid and form key is in
-    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.14"
+    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.21"
```

Comment:
Why 21 vs 20?

Reply:
I had to manually change the flags for only the bladder cohort to include some of the radiation therapy variables.  panc and prostate could be 21 as well.